projects/qwt.cmake

```diff
@@ -0,0 +1,100 @@
+########################################
+# qwt
+########################################
+xpProOption(qwt)
+########################################
+set(QWT_SRC_PATH ${CMAKE_BINARY_DIR}/xpbase/Source/qwt)
+if(WIN32)
+  set(QWT_DL_URL https://sourceforge.net/projects/qwt/files/qwt/6.1.2/qwt-6.1.2.zip/download)
+  set(QWT_DL_MD5 b43a4e93c59b09fa3eb60b2406b4b37f)
+else()
+  set(QWT_DL_URL https://sourceforge.net/projects/qwt/files/qwt/6.1.2/qwt-6.1.2.tar.bz2/download)
+  set(QWT_DL_MD5 9c88db1774fa7e3045af063bbde44d7d)
+endif()
+
+set(PRO_QWT
+  NAME qwt
+  WEB "Qwt" http://http://qwt.sourceforge.net/ "Qwt - Qt Widgets for Technical Applications"
+  LICENSE "LGPL" http://qwt.sourceforge.net/qwtlicense.html "LGPL with exceptions"
+  DESC "The Qwt library contains GUI Components and utility classes which are primarily useful for programs with a technical background."
+  VER 6.1.2
+  DLURL ${QWT_DL_URL}
+  DLMD5 ${QWT_DL_MD5}
+  PATCH ${PATCH_DIR}/qwtconfig.pri.patch  
+)
+########################################
+function(mkpatch_qwt)
+  xpRepo(${PRO_QWT})
+endfunction()
+########################################
+function(download_qwt)
+  xpNewDownload(${PRO_QWT})
+endfunction()
+########################################
+function(patch_qwt)
+  if(NOT (XP_DEFAULT OR XP_PRO_QWT))
+    return()
+  endif()
+
+  xpPatch(${PRO_QWT})
+endfunction()
+########################################
+function(build_qwt)
+  if(NOT (XP_DEFAULT OR XP_PRO_QWT))
+    return()
+  endif()
+
+  if(NOT TARGET qwt)
+    patch_qwt()
+  endif()
+
+  # Make sure the qt5 target this depends on has been created
+  if(NOT (XP_DEFAULT OR XP_PRO_QT5))
+    message(FATAL_ERROR "qwt requires qt5")
+    return()
+  endif()
+
+  if(${XP_BUILD_STATIC})
```

Comment:
(optional) have a variable for static-build to be 'true' or 'false' and use the variable in the add_custom_target instead of duplicating the whole add_custom_target in the if/else.